### PR TITLE
Updates to Bare Metal FAQ for Windows DC

### PR DESCRIPTION
**Pull request by broneal**
added two questions and updated an existing.

### CODE REVIEW

File: Servers/bare-metal-faq.md [hunk 1]
```diff
@@ -101,8 +101,16 @@ No, the storage configurations provisioned for Bare Metal servers are all either
...
...ndows 2012 R2 will automatically activate using [AVMA (Automatic Virtual Machine Activation)](https://technet.microsoft.com/en-us/library/dn303421.aspx) using the host's Hypervisor. Guest VMs running Windows 2008 R2 can activate either using Multi-Activation Keys (MAKs) obtained from Microsoft, or by using the volume license keys available on [Microsoft's site](https://technet.microsoft.com/en-us/library/jj612867.aspx?f=255&MSPPError=-2147217396) along with the CenturyLink KMS host. To use our KMS you will need to [manually specify the KMS host](https://technet.microsoft.com/en-us/library/ff793406.aspx?f=255&MSPPError=-2147217396) on the guest VM. You will need the VM to be on a CenturyLink Cloud customer VLAN to access our KMS host and the IP/port associated, which is 172.17.1.21:1688.  Windows VMs in a Hyper-V environment on Bare Metal servers should use IPs in the same VLAN as configured in the Control portal for the host Bare Metal server.  It is also recommended that IPs be higher in the range so VMs claiming from that same range will not encounter IP conflicts.  For activation of older Windows OSes on guest VMs you will need to obtain your own MAKs and/or activation solution.
```
Comment (chrislittle) [inline Servers/bare-metal-faq.md]: I think we need to provide the commands for the KMS activation and how to validate ports are open (telnet to that tcp port)

```
slmgr /skms 172.17.1.21:1688
```

make the answer easier.....

you should also detail how a customer can claim additional IP's in Control for the Guest VMs.  Until the blueprint feature exists its basically a NOC ticket but we should have a clear statement on how that is done

**Comment by broneal** [comment]
how do these updates look?

**Comment by chrislittle** [comment]
looks good.  i could be picky as most people when they show a command use this model

```
slmgr /skms 172.17.1.21:1688
```

**Comment by broneal** [comment]
good?

**Comment by broneal** [comment]
theeeeeeere we go

**Pull request merged**